templates/macros/disallow.html

```diff
@@ -1,12 +1,12 @@
 {%- macro generate_disallows(disallow, section) -%}
-{%- if disallow or section and section.extra and section.extra.public_draft -%}
-Disallow: {{section.path}}
-{% for s in section.subsections -%}
-{{ self::generate_disallows(disallow=true, section=get_section(path=s)) }}
-{%- endfor -%}
-{%- else -%}
-{%- for s in section.subsections -%}
-{{ self::generate_disallows(disallow=false, section=get_section(path=s)) }}
-{%- endfor -%}
-{%- endif -%}
+  {%- if disallow or section and section.extra and section.extra.public_draft -%}
+    Disallow: {{ section.path }}
+    {% for s in section.subsections -%}
```

Comment:
This should not be formatted, it breaks `robots.txt` generation. A `robots.txt` file should not have any indentation. (I added `{%-` and `-%}` to everything else, so those are fine.)

Reply:
Just removed all indentation for that file; it's a special case that hopefully shouldn't be touched often and I want to be sure this PR doesn't break it.